src/Raven.Server/NotificationCenter/Notifications/Details/LicenseLimitWarning.cs

```diff
@@ -38,9 +38,15 @@ public static void AddLicenseLimitNotification(NotificationCenter notificationCe
                 licenseLimit.Message,
                 AlertType.LicenseManager_LicenseLimit,
                 NotificationSeverity.Warning,
+                key: licenseLimit.Type.ToString(),
```

Comment:
without this the `Id` is just a `LicenseManager_LicenseLimit` (@grisha-kotler please double check). Which means that all license limits alerts will be under the same Id, which is not what we want?